python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -149,13 +149,23 @@ def __init__(
         self._m1m3_actuator_id_index_table: dict[int, int] = dict(
             [(fa[FATABLE_ID], fa[FATABLE_INDEX]) for fa in FATABLE]
         )
+        self._m1m3_actuator_index_id_table: dict[int, int] = dict(
```

Comment:
There is M1M3FATable.actuatorIdToIndex..I can change to snake case and you can use that instead.

Reply:
uhm.. I need to think about that.. I think that implementation will be slower than what I have here no? every time I call `M1M3FATable.actuatorIdToIndex` it will iterate over the entire array. Here I iterate once to create the dictionary and after that I get a simple dictionary lookup.

Reply:
I have plans to better organize cRIOpy bussiness... let's keep that for future.